external/box2d/Box2D/wscript

```diff
@@ -6,95 +6,107 @@ from waflib import Build
 VERSION = '3.1.0'
 APPNAME = 'box2d'
 
+
 def configure(conf):
-    pass
+    print("Configuring Box2D...")
+    plat = conf.env.PLATFORM
+    if not plat:
+        conf.fatal('Pass --platform=...')
 
-def build(bld):
-    box2d = bld.stlib(features = 'c',
-                      includes = '.. include',
-                      source = bld.path.ant_glob('**/*.c'),
-                      target = 'box2d')
+    conf.env.SIMD_CFLAGS, conf.env.SIMD_CXXFLAGS = [], []
+    conf.env.NOSIMD_CFLAGS, conf.env.NOSIMD_CXXFLAGS = [], []
```

Comment:
To do idiomatic Waf, this would be:
```
confirming.env['CFLAGS_SIMD'] = ...
confirming.env['CXXFLAGS_SIMD'] = ...
```

and then you'd use the feature like so:
```python
bld.stlib(features='c',
    use = 'SIMD',
    includes=incs,
    source=srcs,
    target='box2d')
```